packages/commons-ui/.eslintrc.js

```diff
@@ -0,0 +1 @@
+module.exports = require("eslint-config-commons-ui");
```

Comment:
What is the difference between this and eslint commons [config on main](https://github.com/CodeForAfrica/ui/tree/main/packages/eslint-config-commons-ui)?

Reply:
ooohhh, never mind. I got it